README.md

```diff
@@ -3,31 +3,30 @@
 [![CircleCI](https://circleci.com/gh/HDE/arsenic/tree/master.svg?style=svg)](https://circleci.com/gh/HDE/arsenic/tree/master) [![Documentation Status](https://readthedocs.org/projects/arsenic/badge/?version=latest)](http://arsenic.readthedocs.io/en/latest/?badge=latest)
 
 
-Asynchronous, framework-independent webdriver client for AsyncIO/Aiohttp and
-Tornado.
+Asynchronous webdriver client built on asyncio.
 
 
 ## Quickstart
 
-Let's run Firefox via Geckodriver using Aiohttp.
+Let's run a local Firefox instance.
 
 
 ```python
```

Comment:
I've just completely re-wamped the documentation, which should the one true source of information. But whenever I check out a library on github, I like to see a few lines of code of what using that library looks like.